pkg/hostagent/pods.go

```diff
@@ -191,7 +192,21 @@ func (agent *HostAgent) syncEps() bool {
 				if ep.Uuid != epidstr {
 					continue
 				}
-
+				agent.indexMutex.Lock()
```

Comment:
Should this processing happen in code block L146-L152?

Reply:
yes